fashrevwall/models/models.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+
+class Tweet(models.Model):
+    text = models.CharField(max_length=200)
```

Comment:
I don't think we need this, as they have only requested the username and the image.